src/main/resources/db/migration/V2018.07.26__Initialize_database.sql

```diff
@@ -0,0 +1,65 @@
+CREATE TABLE problem_set(
```

Comment:
this migration versioning is not good, since we can have multiple migrations added on the same day. Either use the entire timestamp or just do the Flyway default way of using an integer V1, V2, V3

Reply:
Ok, I'll use an integer